Swipeable.js

```diff
@@ -200,8 +226,12 @@ export default class Swipeable extends Component {
   };
 
   _currentOffset = () => {
-    const { leftWidth = 0, rowWidth = 0, rowState } = this.state;
-    const { rightOffset = rowWidth } = this.state;
+    const { rowState } = this.state;
+    const leftWidth = this.state.leftWidth ? this.state.leftWidth : 0;
+    const rowWidth = this.state.rowWidth ? this.state.rowWidth : 0;
+    const rightOffset = this.state.rightOffset
```

Comment:
Why changing this? Seems like flow can deal with this syntax, can it?

Also: this is not completely accurate with the way this code used to work. Instead of doing:
```this.state.foo ? this.state.foo : bar```
this should rather be: 
```this.state.foo === undefined ? this.state.foo : bar```

For most of the cases it won't matter but it will change behavior in the case when `this.state.rightOffset = 0`. Then, instead of having `rightOffset = 0` we will get `rightOffset = rowWidth`.

Reply:
You're absolutely right, this is broken for the 0 case. Good catch, sorry I missed that.

The reason I changed it in the first place was because I was typing `leftWidth`, `rowWidth`, etc. as nullable, which meant they could be either `null` or `undefined`, but the "destructuring assignment with default value" syntax only handles `undefined`.

I'll change the types to be `number | typeof undefined` instead of `?number` and switch the logic back to the destructuring assignment.